source/fundamentals/enterprise-auth.txt

```diff
@@ -93,19 +85,32 @@ preferences:
            AuthMechanism: "GSSAPI",
            Username:      "<username>",
            Password:      "<password>",
+           PasswordSet:   true,
    }
 
-   uri := "mongodb://<hostname>:<port>"
+   uri := "<connection uri>"
    clientOpts := options.Client().ApplyURI(uri).SetAuth(credential)
    
    client, err := mongo.Connect(context.TODO(), clientOpts)
 
-Configuration Options
-~~~~~~~~~~~~~~~~~~~~~
+You don't need to define a password or the ``PasswordSet`` field in
+your ``Credential`` struct if you store authentication keys in
+``keytab`` files. You can initialize a credential cache for
```

Comment:
Question: Could you explain why I might initialize a credential cache in the docs? Does it affect the driver behavior in any way? If so, it would be useful to document. If not, I think maybe it would be better to omit.